extensions/wikia/Forum/ForumController.class.php

```diff
@@ -113,21 +103,31 @@ public function getWallForIndexPage( $title ) {
 
 	public function boardNewThread() {
 		parent::newMessage();
-		$this->isTopicPage = $this->getVal( 'isTopicPage', false );
-		if ( $this->isTopicPage ) {
+		$isTopicPage = $this->getVal( 'isTopicPage', false );
+		$this->response->setVal( 'isTopicPage', $isTopicPage );
+
+		if ( $isTopicPage ) {
 			$forum = new Forum();
 
 			$list = $forum->getBoardList();
 
-			$this->destinationBoards = [ [ 'value' => '', 'content' => wfMessage( 'forum-board-destination-empty' )->escaped() ] ];
+			$destinationBoards = [
+				[
+					'value' => '',
+					'content' => wfMessage( 'forum-board-destination-empty' )->escaped()
+				]
+			];
 
 			foreach ( $list as $value ) {
-				$this->destinationBoards[] = [ 'value' => htmlspecialchars( $value['name'] ), 'content' => htmlspecialchars( $value['name'] ) ];
+				$destinationBoards[] = [
+					'value' => htmlspecialchars( $value[ 'name' ] ),
+					'content' => htmlspecialchars( $value[ 'name' ] )
+				];
 			}
+
+			$this->response->setVal( 'destinationBoards', $destinationBoards );
 		}
 
-		// MAIN-7865: Don't render MiniEditor on Monobook
-		$this->response->setVal( 'showMiniEditor', $this->wg->EnableMiniEditorExt && $this->app->checkSkin( 'oasis' ) );
```

Comment:
I think it is an error. MiniEditor is actively used and lack of the value would cause any checks evaluate to `false` unless a default is set.

UPDATE: I don't see any default to be set anywhere. Only `if ($showMiniEditor)` checks.